desc/objectives/linear_objectives.py

```diff
@@ -2566,3 +2562,631 @@ def compute(self, Psi, **kwargs):
     def target_arg(self):
         """str: Name of argument corresponding to the target."""
         return "Psi"
+
+
+class FixR(_Objective):
```

Comment:
Wouldn't this (any the similar new constraints) be more useful if you could select which modes to fix, or pass "True" or something to fix all modes, like `FixBoundaryR`?

Reply:
These classes were redundant with `FixModesR` and `FixModesZ` and have been removed. Made a note in the code to add something similar for lambda.